chia/full_node/full_node.py

```diff
@@ -758,7 +766,7 @@ def _close(self):
     async def _await_closed(self):
         for task_id, task in list(self.full_node_store.tx_fetch_tasks.items()):
             cancel_task_safe(task, self.log)
-        await self.connection.close()
+        await self.db_wrapper.close()
```

Comment:
yeah. This is consistent with the current behavior. Maybe it would make more sense if we would wrap the creation of the write connection too, in a create-function.